### PR TITLE
Add SES setup for #33

### PR DESCRIPTION
**Pull request by nelsonic**
This pull request creates _detailed_ instructions for how to setup AWS SES for sending email! 🚀 
+ [x] Add SES email verification setup instructions #33

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles here's another PR improving the documentation for setting up AWS SES for sending email. 💌 

Please review and merge when you can. 
Thanks! 🌻

**Comment by nelsonic** [comment]
Read it here: https://github.com/dwyl/learn-amazon-web-services/blob/aws-ses-setup-instructions-issue%2333/ses.md

File: ses.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+and escaping the "sandbox".
+
+
+## Why? 🤷
+
+All Software as a Service (SaaS) Products that create value send **`email`**. <br />
+Even if it's just a reminder to pay a bill,
+an **`email`** has to be sent _reliably_. <br />
+Amazon's SES is a _reliable_, full featured
+and _~~cheap~~_ _cheapest_ email service.
```
Comment (iteles) [inline ses.md:14]:
```suggestion
and _~~cheap~~_ _the cheapest_ email service.
```

File: ses.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+In the "Email Addresses" menu, click on "Verify a New Email Address":
+
+![aws-ses-add-email-address](https://user-images.githubusercontent.com/194400/74105174-59d93f80-4b53-11ea-8852-274b7d4f9028.png)
+
+
+> **Note**:
+In our case we had to _setup_ a **_free_ Zoho Mail** email account
+for our domain in order to _receive_ the confirmation email.
+See:
+[issues/35](https://github.com/dwyl/learn-amazon-web-services/issues/35)
```
Comment (iteles) [inline ses.md:78]: 👍

**Review by iteles — APPROVED** [review]
@nelsonic I made one tiny suggestion but not too fussed about it, so will merge this in half an hour either way ☺️

File: ses.md (hunk 1 above)
Comment (nelsonic) [reply ses.md:14]: this would have been a good suggestion ... why did you merge without it?

**Pull request merged**